test/unit/spec/v1/EnrollCall_spec.js

```diff
@@ -553,6 +550,9 @@ Expect.describe('EnrollCall', function() {
   }
 
   describe('Header & Footer', function() {
+    beforeEach(() => {
+      jest.setTimeout(20000);
+    });
```

Comment:
I ran this `describe` block locally on its own with this removed and didn't see any test timeouts. The tests each seem to take about 1000-2000ms to run, not 20,000. Is there something about running these concurrently on Bacon that causes timeouts?

Reply:
Let me give this a try, this was apart of a commit I cherry picked from someone else's rotation, I will remove this if it is not needed 🙏